scripts/repro_matrix/generate_html_miracl.py

```diff
@@ -159,6 +159,9 @@ def generate_table_rows(table_id, split, metric):
 
                 for expected in splits['scores']:
                     for metric in expected:
+                        if str(expected[metric])[-1] == "5":
```

Comment:
you can use Python's `isclose()` for comparison or use f-string format to do proper rounding.

See what I did in the mrtydi regressions.